thirdeye-persistence/src/test/java/ai/startree/thirdeye/datalayer/bao/TestAnomalyManager.java

```diff
@@ -183,19 +185,48 @@ public void testFeedback() {
     Assert.assertEquals(mergedResult1.getFeedback().getFeedbackType(), AnomalyFeedbackType.ANOMALY);
   }
 
+  @Test
+  public void testGetAllAnomaliesForFeedback() {
```

Comment:
Let's have separate test methods for `findAll` and `filter`

Reply:
Done